src/database/models/Youfocus/UserPlaylist.ts

```diff
@@ -2,6 +2,17 @@ import { Schema, model, models, Model } from 'mongoose';
 import { UserPlaylistModel } from '@/interfaces';
 import { databaseModels } from '@/constant';
 
+
+// Schema to track total time watched for the playlist
+const TimeSchema = new Schema(
+  {
+    minutes: { type: Number, default: 0 },
```

Comment:
Don't create a new Schema. Just simply add a key in User Platlist as learningTime

Reply:
Or timeWatched as you've doe

Reply:
Don't do it for Seconds. Just take minutes only